pof/src/parse.rs

```diff
@@ -1341,9 +1341,11 @@ impl<'a> ParseCtx<'a> for DaeContext<'a> {
                     self.normal_id = ctx.normal_ids[index as usize];
                 }
                 fn add_texcoord(&mut self, _: &VertexContext, reader: &SourceReader<'a, ST>, index: u32, set: Option<u32>) {
-                    assert!(set.map_or(true, |set| set == 0));
-                    let [u, v] = reader.get(index as usize);
-                    self.uv = (u, v);
+                    // maybe raise an import warning over this?
+                    if set == Some(0) {
```

Comment:
I think `set == None` can also happen (the assert allows it) and is treated the same as `set == Some(0)`. Or should we assert that it is not none?

Reply:
This gets called regardless for all vertices or whatever, right? Given we explicitly handle untextured, we should expect this to happen occasionally and should be allowed, and any warnings should only concern `Some(non-zero)`.

Reply:
Well, the current code is now incorrect in the case `set == None`. If you want to preserve the behavior from before you should use `set.map_or(true, |set| set == 0)` as the if condition.

Reply:
Oh, I understand now. It seem then it should be left alone, if none? There is no texcoord to grab, but apparently it worked? Either it seems like it shouldn't matter for None anyway. It's either getting nothing useful, or potentially overwriting the 0th texcoords, for 1 or more, so I think this is still the right approach.

Reply:
Oh right, if `None` means it is untextured then of course there is no point in loading the uv. (It's not completely obvious to me that this is the case, but I'll take your word for it.)

Reply:
As long as its initialized to something safe I think it should be ok. I'm not certain this is the case, but this is rust so I'm it is.